airbyte-bootloader/build.gradle

```diff
@@ -11,19 +26,42 @@ dependencies {
     implementation project(':airbyte-protocol:protocol-models')
     implementation project(':airbyte-persistence:job-persistence')
 
-    implementation libs.temporal.sdk
-    implementation libs.flyway.core
+    testAnnotationProcessor platform(libs.micronaut.bom)
+    testAnnotationProcessor libs.bundles.micronaut.test.annotation.processor
```

Comment:
Out of curiosity, what happened to `libs.flyway.core`?

Reply:
It got moved up in this block to ensure that we get the correct version.